backend/Application/Api/Rest/ExportController.cs

```diff
@@ -42,21 +45,24 @@ public async Task<ActionResult> GetStatementExport(string accountAddress, DateTi
 
         var query = dbContext.AccountStatementEntries
             .AsNoTracking()
-            .Where(x => x.AccountId == account.Id);
+            .OrderByDescending(x => x.Timestamp)
```

Comment:
Why do you have to order this now when it wasn't needed previously?

Reply:
Because otherwise it orders by index, and then it doesn't use the index I added.